parcel.go

```diff
@@ -14,47 +14,91 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
+	res, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) VALUES (:client, :status, :address, :createdAt)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("createdAt", p.CreatedAt))
 	// верните идентификатор последней добавленной записи
-	return 0, nil
+	if err != nil {
+		return -1, err
+	}
+	identifier, err := res.LastInsertId()
+	if err != nil {
+		return -1, err
+	}
+	return int(identifier), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
 	// реализуйте чтение строки по заданному number
 	// здесь из таблицы должна вернуться только одна строка
-
+	row := s.db.QueryRow("SELECT number, client, status, address, created_at FROM parcel WHERE number = :number",
+		sql.Named("number", number))
 	// заполните объект Parcel данными из таблицы
 	p := Parcel{}
-
+	err := row.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+	if err != nil {
+		return p, err
```

Comment:
Лучше нулевое значение отправлять
```suggestion
		return Parcel{}, err
```